src/java_bytecode/library/CMakeLists.txt

```diff
@@ -4,16 +4,13 @@ set(CMAKE_JAVA_COMPILE_FLAGS -sourcepath "src" -d "classes" -XDignore.symbol.fil
 file(GLOB_RECURSE java_sources "*.java")
 set(JAR_NAME "core-models")
 add_jar(${JAR_NAME} ${java_sources})
-get_filename_component(CORE_MODELS_JAR "core-models.jar" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+get_filename_component(CORE_MODELS_JAR "core-models.jar" REALPATH BASE_DIR ${CMAKE_CURRENT_BINARY_DIR})
 get_filename_component(JAVA_CORE_MODELS_INC "java_core_models.inc" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Should JAVA_CORE_MODELS_INC also use CMAKE_CURRENT_BINARY_DIR here as well? If I'm reading the Cmake correctly, then JAVA_CORE_MODELS_INC is a generated file, so it'd be nicer if that was generated into the build directory rather than the source directory - but maybe that just opens a bigger can of worms than this PR is fixing?

Reply:
Yes, and yes. The ideal approach would generate it somewhere, then add that path to the consumer library's include search path. However that's another half-day's cmake studies that I don't have the appetite for just now, so let's address that separately ;)

Reply:
Understood, and I don't blame you :-)